activemodel/lib/active_model/validations/comparison.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module ActiveModel
+  module Validations
+    class ComparisonValidator < EachValidator # :nodoc:
+      include Comparability
+
+      def check_validity!
+        unless (options.keys & COMPARE_CHECKS.keys).any?
+          raise ArgumentError, "Expected one of :greater_than, :greater_than_or_equal_to, "\
+          ":equal_to, :less_than, :less_than_or_equal_to, nor :other_than supplied."
```

Comment:
@ChaelCodes Typo: `nor` -> `or`

I would also add `option` in the exception message like: `... or :other_than option supplied`.